src/librustdoc/html/format.rs

```diff
@@ -788,14 +797,14 @@ fn fmt_type(t: &clean::Type, f: &mut fmt::Formatter, use_absolute: bool,
                 _ => {
```

Comment:
I'm not sure it'd be better. But don't hesitate to do it if you think it can be improved.